scripts/dev_setup.sh

```diff
@@ -147,17 +147,16 @@ function install_protoc {
   )
   rm -rf "$TMPFILE"
 
-  # TODO(larry): uncomment when upstream issue is resolved.
-  # # Install the cargo plugins
-  # if ! command -v protoc-gen-prost &> /dev/null; then
-  #   cargo install protoc-gen-prost
-  # fi
-  # if ! command -v protoc-gen-prost-serde &> /dev/null; then
-  #   cargo install protoc-gen-prost-serde
-  # fi
-  # if ! command -v protoc-gen-prost-crate &> /dev/null; then
-  #   cargo install protoc-gen-prost-crate
-  # fi
+  # Install the cargo plugins
+  if ! command -v protoc-gen-prost &> /dev/null; then
+    cargo install protoc-gen-prost --locked
```

Comment:
can we add `--version` flag to pin the version?

Reply:
The issue is from indirect dependency; pinning to a specific version doesn't help us avoid the issue tho.